lib/cylc/review.py

```diff
@@ -843,7 +836,7 @@ def _check_file_path(cls, path):
         cls._check_path_normalised(path)
         # Get rootdir and sub-path.
         head, tail = os.path.split(path)
-        while os.path.split(head)[0]:
+        while os.path.dirname(head) not in ['', '/']:
```

Comment:
```suggestion
        while os.path.dirname(head) not in ['', os.sep]:
```

Reply:
Either solution is wrong. Need constant for root directory!

Reply:
Done your way.

Reply:
> Either solution is wrong

If we want to be *really* pedantic: `os.path.abspath(os.sep)`